HIRS_Utils/src/main/java/hirs/tpm/TPMBaselineGenerator.java

```diff
@@ -591,16 +591,15 @@ private void parseBaseline(final String baselineName, final InputStream inStream
             throws IOException, TPMBaselineGeneratorException {
         BufferedReader reader = new BufferedReader(new InputStreamReader(
                 inStream, "UTF8"));
-        String dataRow = reader.readLine();
+        String dataRow = "";
         HashMap<TPMBaselineFields, String> fieldMap = new HashMap<>();
 
         try {
-            while (reader.ready()) {
-                if (StringUtils.isBlank(dataRow)) {
-                    dataRow = reader.readLine();
-                    continue;
+            while (true) {
```

Comment:
As part of the work on this ticket, could you do me a favor and go into the `TPMBaselineGeneratorTest` class and add an `@Test` annotation to 3 tests that are currently not running and make sure they all pass? Some of those apply to these changes, so it would be good to ensure continuing test coverage on this.

They are `generateBaselineFromCSVFileContainsInvalidRecords`, `generateBaselineFromCSVFileNullBaselineName`, and `generateBaselineFromCSVFileContainsNoRecords`.

Reply:
Added.  `generateBaselineFromCSVFileContainsNoRecords` did not catch an exception because of the new regex check.  I added an isEmpty() method to the TPMBaseline class so that an empty csv file that generates an empty TPMBaseline object is differentiated from a valid csv file with an empty last line.